Bleedify/Bleedify.Tests/DonatorRepositoryTests.cs

```diff
@@ -0,0 +1,178 @@
+﻿using BleedifyModels.ModelsEF;
+using BleedifyModels.Repositories;
+using BleedifyModels.Validators;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bleedify.Tests
+{
+    [TestClass]
+    public class DonatorRepositoryTests
+    {
+        #region Initialization and Cleanup
+
+        private static DonatorRepository _donatorRepository;
+
+
+        [ClassInitialize]
+        public static void InitializeTest(TestContext context)
+        {
+            _donatorRepository = new DonatorRepository(new DonatorValidator());
+        }
+
+
+        [ClassCleanup]
+        public static void CleanUpTest()
+        {
+            _donatorRepository.GetAll().ToList().ForEach(x =>
+            {
+                if (String.Compare(x.Nume, "TEST", StringComparison.Ordinal) == 0 ||
+                    String.Compare(x.Nume, "TESTUPDATE", StringComparison.Ordinal) == 0)
+                {
+                    _donatorRepository.Delete(x.Id);
+                }
+            });
+        }
+
+        #endregion
+
+        #region AddTest
+
+        [TestMethod]
+        public void AddDonatorTest()
+        {
+            var _datetime = new DateTime(2018, 4, 14);
+            var _donator = new Donator();
+            _donator.Id = 1;
+            _donator.IdUtilizator = 1;
+            _donator.Nume = "TEST";
+            _donator.Prenume = "TEST";
+            _donator.DataDonarePosibila = _datetime;
+
+            var _initialSize = _donatorRepository.GetAll().Count();
+
+            try
+            {
+                _donatorRepository.Add(_donator);
+            }
+            catch (Exception e)
+            {
+                Assert.Fail();
+            }
+
+            Assert.IsTrue(_donatorRepository.GetAll().Count() == _initialSize + 1);
+        }
+
+        #endregion
+
+        #region FindTest
+
+        [TestMethod]
+        public void FindDonatorTest()
+        {
+            var _datetime = new DateTime(2018, 4, 14);
+            var _donator = new Donator();
+            _donator.Id = 1;
+            _donator.IdUtilizator = 2;
+            _donator.Nume = "TEST";
+            _donator.Prenume = "TEST";
+            _donator.DataDonarePosibila = _datetime;
+
```

Comment:
De cat sa faci n variabile maib ine iti declari local un doantor nou asa:
``var donatorTest = new Donator
{ 
Nume = blabla....,
Prenume = blabla...
};``

Acum poti sa lasi si asa, dar e mai curat codul cand folosesti contructorul clasei de cat sa faci variabile in plus.

Reply:
am facut-o ca sa pot sa o compar mai incolo fara sa mai scrie new DateTime(2018, 04, 14)